zest/releaser/bzr.py

```diff
@@ -24,10 +24,10 @@ def name(self):
         return dir_name
 
     def available_tags(self):
-        tag_info = execute_command('bzr tags')
-        tags = [line[:line.find(' ')] for line in tag_info.split('\n')]
+        tag_info = execute_command([u'bzr', u'tags'])
+        tags = [line[:line.find(u' ')] for line in tag_info.split(u'\n')]
         tags = [tag for tag in tags if tag]
-        logger.debug("Available tags: %r", tags)
+        logger.debug(u"Available tags: {0!r}".format(tags))
```

Comment:
I don't know what is standard in 3.4. I'm still used to %s :-) 

Could you provide a link or so? I might have to change my preference.